behind-the-veil-siteroot/imports/ui/components/card/BookingCard.jsx

```diff
@@ -139,6 +138,26 @@ export const BookingCard = ({
                     </Button>
                 );
                 break;
+            case BookingStatus.CONFIRMED:
```

Comment:
since both statements map to the exact same logic, you can get rid of code duplication by doing:
```
            case BookingStatus.CONFIRMED:
            case BookingStatus.OVERDUE:
                // if a booking is confirmed, add "cancel" button
                additionalButtons.push(
                    <Button className={purpleButtonClass}
                        onClick={() => navigateTo('/cancel-booking/' + bookingId)}>
                        <XCircleIcon className="icon-base"/>
                        Cancel
                    </Button>
                );
                break;
```
instead of duplicating the logic

Reply:
Done